CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ Fourth digit in release number means PC Engines patch.
 
 ## [Unreleased]
 
+## [rel-1.11.0.2] - 2017-12-22
+### Fixed
+- bug with serial console printing
```

Comment:
@miczyg1 where is code ?

Reply:
@pietrushnic changes were already merged here: https://github.com/pcengines/seabios/pull/13

Reply:
@miczyg1 ok